src/components/calculator/PointsDisplay.tsx

```diff
@@ -18,6 +18,7 @@ export interface RiskLevel {
   title: string
   max: number
   icon?: IconType
+  override?: boolean
```

Comment:
Could you use a more descriptive variable name? Overrides what?

Possibly cleaner to use `overrideThermomemterStyle: string` and then put the name of the style here.